event.yml

```diff
@@ -1,16 +1,14 @@
-#
-# Event fock
-#
-
-# admin page event
-eccube.event.route.admin_order.response:
-    - [onResponseAdminOrderPdfBefore, NORMAL]
-eccube.event.route.admin_order_page.response:
-    - [onResponseAdminOrderPdfBefore, NORMAL]
-
-# deprecated event (for v3.0.0 - 3.0.8)
-eccube.event.render.admin_order.before:
-    - [onRenderAdminOrderPdfBefore, NORMAL]
-eccube.event.render.admin_order_page.before:
-    - [onRenderAdminOrderPdfBefore, NORMAL]
-
+#
+# Event fock
+#
+
+# New hookpoint for version >= 3.0.9
+Admin/Order/index.twig:
+    - [onAdminOrderRender, NORMAL]
```

Comment:
"index" not means "default page" (like index.html).
it means ... "index search" "index card" "index page" ...index.

i'd like to rename to `onAdminOrderIndexRender` to clarify.